streaming/src/test/scala/org/apache/spark/streaming/Demo.scala

```diff
@@ -0,0 +1,94 @@
+/*
```

Comment:
Yes, I will remove this file and add some unit tests if people agree that this PR is in the right direction. For now, this temp application should be enough to demonstrate the usage.